src/test/signin.test.js

```diff
@@ -0,0 +1,233 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import bcrypt from 'bcrypt';
+
+import models from '../db/models';
+import app from '../index';
+
+chai.use(chaiHttp);
+
+const { expect } = chai;
+const signinUrl = '/api/v1/auth/signin';
+const { User } = models;
+
+const testUserDetails = {
+  first_name: 'Olawumi',
+  last_name: 'Yusuff',
+  email: 'user@test.com',
+  password: '12345678'
+};
+let testUser;
+
+
+before((done) => {
```

Comment:
I think a cleaner way would be to seed data using Sequelize. 
Also, you could pre-hash passwords. ie, hash a password, say 'pass', console.log it and keep it for future use. 
Bycrypt is quite heavy. If you hash multiple times, it will slow down your tests.